src/components/ImportForm/submit-utils.ts

```diff
@@ -93,7 +94,11 @@ export const createResources = async (
 
   let createdComponent;
   if (showComponent) {
-    await createSecrets(importSecrets, workspace, namespace, true);
+    const allSecrets = await getSecretResource(namespace);
+    const secretsToCreate = importSecrets.filter((secret) =>
```

Comment:
is it secure to fetch all secrets and than filter them?

Reply:
The description was update to `should not create application but create components without secrets`